bin/docker-entrypoint

```diff
@@ -42,6 +42,9 @@ for group_id in $LOCAL_GROUP_IDS; do
 
 done
 
+# Set ownership to KubeNow folder
+chown -R "$username" "/opt/KubeNow"
```

Comment:
Set ownership of KubeNow folder inside image to current user (to avoid sudo inside provisioner image)